rfc6962-bis.xml

```diff
@@ -510,186 +596,170 @@ Section 8.2 of <xref target="RFC3447"/>) using a key of at least 2048 bits.
         </t>
         <t>
           The encoding of the digitally-signed element is defined in <xref target='RFC5246'/>.
         </t>
         <t>
-          <spanx style="verb">sct_version</spanx> is the version of the
-protocol to which the SCT conforms. This version is v2. Note that <xref target="RFC6962">SignedCertificateTimestamp v1</xref> had a different definition of <spanx style="verb">signed_entry</spanx>.
-        </t>
-        <t>
-          <spanx style="verb">log_id</spanx> is the DER encoding of an OID, excluding the ASN.1 tag and length bytes, that the log operator has allocated for the purpose of uniquely identifying this log. Note that the ASN.1 length and the opaque vector length are identical in size (1 byte) and value, so the DER encoding of the OID can be reproduced simply by prepending an OBJECT IDENTIFIER tag (0x06) to the opaque vector length and contents. This OID is specified in the log's metadata.
+          <spanx style="verb">timestamped_entry</spanx> is a <spanx style="verb">TransItem</spanx> structure that MUST be of type <spanx style="verb">x509_entry</spanx> or <spanx style="verb">precert_entry</spanx> (see <xref target="tree_leaves"/>) and MUST have an empty <spanx style="verb">item_extensions</spanx> vector.
         </t>
+      </section>
+      <section title="Merkle Tree Head" anchor="tree_head">
+        <figure>
+          <preamble>
+            The log stores information about its Merkle Tree in a <spanx style="verb">TransItem</spanx> structure of type <spanx style="verb">tree_head</spanx>, which in this version (v2) encapsulates a <spanx style="verb">TreeHeadDataV2</spanx> structure:
+          </preamble>
+          <artwork>
+    opaque NodeHash[HASH_SIZE];
+
+    struct {
+        uint64 timestamp;
+        uint64 tree_size;
+        NodeHash root_hash;
+        SthExtension sth_extensions&lt;0..2^16-1&gt;;
+    } TreeHeadDataV2;</artwork>
+        </figure>
         <t>
-          <spanx style="verb">timestamp</spanx> is the current <xref target="RFC5905">NTP Time</xref>, measured since the epoch (January 1, 1970, 00:00), ignoring leap seconds, in milliseconds.
+          <spanx style="verb">timestamp</spanx> is the current <xref target="RFC5905">NTP Time</xref>, measured in milliseconds since the epoch (January 1, 1970, 00:00), ignoring leap seconds.
         </t>
         <t>
-          <spanx style="verb">entry_type</spanx> may be implicit from the context in which the SCT is presented.
+          <spanx style="verb">tree_size</spanx> equals the number of entries currently in the log's Merkle Tree.
```

Comment:
Addressed in https://github.com/robstradling/certificate-transparency-rfcs/commit/5a0658ff5b674b30745c2021ebb8c44488582db4